cpp/tests/groupby/groupby_test_util.hpp

```diff
@@ -128,5 +131,57 @@ inline void test_single_scan(column_view const& keys,
     expect_vals, *result.second[0].results[0], debug_output_level::ALL_ERRORS);
 }
 
+template <typename T>
+inline T frand()
```

Comment:
We have utilities explicitly to avoid having to write code like this: https://github.com/rapidsai/cudf/blob/ba2cbd91348b18e1685b6149efe73e3f2aeef9ec/cpp/include/cudf_test/base_fixture.hpp#L121

Reply:
Due to the nature of the tests as they are right now (particularly the aggregations, where I'm testing against explictly expected values) this would be a huge change and I'd have to regenerate and verify (against arrow, by hand) all my expected results.